### ISSUE TITLE
XBB.1.5 ACE2 binding correlation to BA.2 and yeast DMS

### ISSUE THREAD

**Issue by Bernadetadad** [dms-vep/SARS-CoV-2_XBB.1.5_spike_DMS#75]
@jbloom one reviewer is asking for explanation why correlation between yeast DMS and XBB.1.5 ACE2 binding is significantly worse than that between yeast DMS and BA.2 ACE2 binding. 

I think it has something to do with XBB.1.5 RBD itself, specifically, I wonder if somehow XBB.1.5 RBD is more "sensitive" to the binding changes caused by RBD movement. The reason I wonder this is because XBB.1.5 and BA.2 library measurements don't have great correlation, but XBB.1.5 full spike and XBB.1.5 RBD only library binding correlated really well. I also think that a lot of sites that are off diagonal in these correlation plots are the ones that we would think are involved in modulating RBD movement - which would explain why correlation with yeast data is not great and if my hypothesis is correct would also explain why correlation with BA.2 library is not great.

To that end, I wonder if we could make a correlation plots between XBB.1.5 full spike library ACE2 binding and BA.2/yeast ACE2 binding and colour the point by R values for ACE2 binding vs serum escape? This might need to be filtered to show only sites that has some minimum number of mutations measured otherwise it may be quite noisy.

**Comment by jbloom** [comment]
As we discussed, a better way to do this may be to separate RBD into ACE2 proximal versus distal sites. Are you able to make a stratification of sites along those lines?

**Comment by Bernadetadad** [comment]
yes, I think we have a notebook that does it already so I can figure it out from there.

**Comment by Bernadetadad** [comment]
Closing as it's solved with #81
